helm/templates/olm/scorecard.yaml

```
@@ -1,50 +1,51 @@
-- op: add
-  path: /stages/0/tests/-
-  value:
-    entrypoint:
+{{ if .Values.olm }}
+apiVersion: scorecard.operatorframework.io/v1alpha3
+kind: Configuration
+metadata:
+  name: config
+stages:
+- parallel: true
+  tests:
+  - entrypoint:
+    - scorecard-test
+    - basic-check-spec
+    image: quay.io/operator-framework/scorecard-test:v1.22.0
```

```suggestion
    image: quay.io/operator-framework/scorecard-test:v1.22
```
This probably doesn't hurt :)